about.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css" />
+    <script
+      src="https://kit.fontawesome.com/38b6d4ab77.js"
+      crossorigin="anonymous"
+    ></script>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <script src="./jsScripts/menuModal.js" defer></script>
+    <title>Programming Contest</title>
+</head>
+<body>
+    <header>
+        <nav class="top-nav">
+          <ul class="nav-list">
+            <li>
+              <a href="#" class="nav-item"><i class="fa fa-facebook"></i></a>
+            </li>
+            <li>
+              <a href="#" class="nav-item"><i class="fa fa-twitter"></i></a>
+            </li>
+            <li><a href="#" class="nav-item">English</a></li>
+            <li><a href="#" class="nav-item">My Page</a></li>
+            <li><a href="#" class="nav-item">Logout</a></li>
+          </ul>
+        </nav>
+  
+        <nav class="main-nav">
+          <a href="./index.html" class="logo"
+            ><img src="./img/icpc-logo.png" alt="ICPC-Logo"
+          /></a>
+          <ul class="main-nav-list nav-list">
+            <li><a href="#" class="main-nav-item nav-item" id="close-menu">X</a></li>
+            <li><a href="./index.html" class="main-nav-item nav-item">Home</a></li>
+            <li><a href="#logo-section" class="main-nav-item nav-item">Logo Detail</a></li>
+            <li><a href="#past-section" class="main-nav-item nav-item">Past Events</a></li>
+            <li><a href="#partner" class="main-nav-item nav-item">Sponser</a></li>
+            <li><a href="#" class="main-nav-item nav-item">News</a></li>
+            <li><a href="#" class="nav-item nav-btn">Campaign</a></li>
+          </ul>
+        </nav>
+      </header>
+
+      <main>
+        <section class="intro-section">
+            <div class="container">
+                <a href="#" class="menu-icon"><i class="fa-solid fa-bars"></i></a>
+                <div class="about">
+                    <h2 class="intro-preheading">"International Conference LLC"</h2>
+                    <h1 class="main-heading">
+                      International Conference LLC
+                    </h1>
```

Comment:
[OPTIONAL] Please grab all this text here inside a heading `h1` tag for semantic tag purposes. :+1: